client/me/purchases/cancel-privacy-protection/index.jsx

```diff
@@ -70,8 +71,7 @@ const CancelPrivacyProtection = React.createClass( {
 			return true;
 		}
 
-		const { selectedSite } = props,
-			purchase = getPurchase( props );
+		const { selectedSite } = props, purchase = getPurchase( props );
```

Comment:
Would be nice. I've been moving to one `const` per variable though, which works around it. so

```
const { selectedSite } = props;
const purchase = getPurchase( props );
```